docs/storage_snippets.py

```diff
@@ -138,7 +139,8 @@ def delete_blob(client, to_delete):
     from google.cloud.exceptions import NotFound
     client = storage.Client()
     bucket = client.get_bucket('my-bucket')
-    assert isinstance(bucket.list_blobs(), list)
+    blobs = list(bucket.list_blobs())
+    assert isinstance(blobs, list)
```

Comment:
Not sure what makes sense. Probably more changes than are worth it in this PR? Definitely need a "local" bucket name so hard-coding a string in the doc is bad.

I would say don't sweat it in this PR?